File: Python-Programing-Basics/Nested Loops/Nested Loops - Lab/04_sum_of_two_numbers/sum_of_two_numbers.py
start = int(input())
ending = int(input())
search_number = int(input())
counter = 0
for i in range(start, ending + 1):
    for i2 in range(start, ending + 1):
        counter += 1
        if i + i2 == search_number:
            print(f"Combination N:{counter} ({i} + {i2} = {search_number})")
            exit()

print(f"{counter} combinations - neither equals {search_number}")

File: JavaScript/Programming Basics - JavaScript/Nested Loops/sumOfTwoNumbers.js
function sumOfTwoNums(input){
    let start = Number(input[0]);
    let end = Number(input[1]);
    let result = Number(input[2]);
    let counter = 0;
    for (let i = start; i <= end; i++){
        for (let j = start; j <= end; j++){
            counter += 1
            if (i + j === result){
                console.log(`Combination N:${counter} (${i} + ${j} = ${i + j})`);
                return;
            }
        }
    }
    console.log(`${counter} combinations - neither equals ${result}`);
}


sumOfTwoNums(["88",
"888",
"1000"]);